release/models/platform/openconfig-platform.yang

```diff
@@ -436,6 +443,47 @@ module openconfig-platform {
         corresponding subcomponent reference from the parent
         component.";
     }
+
+    leaf redundant-role {
+      type oc-platform-types:component-redundant-role;
+      description
+        "For components that have redundant roles (e.g. two
+        supervisors in a device, one as primary the other as secondary),
+        this reports the role of the component.";
+    }
+
+    container last-switchover-reason {
+      description
+        "For components that have redundant roles (e.g. two
+        supervisors in a device, one as primary the other as secondary),
+        this reports the reason of the last change of the
+        component's role.";
+
+      uses oc-platform-types:component-redundant-role-switchover-reason;
+    }
+
+    leaf last-switchover-time {
+      type oc-yang:date;
```

Comment:
Can we make this be `oc-types:timeticks64` such that it is expressed as nanoseconds since the Unix epoch? The aim here is try and standardise on this format for timestamps through the models.

Reply:
Thanks. Done!